p2panda-core/src/operation.rs

```diff
@@ -128,6 +128,32 @@ where
     pub fn hash(&self) -> Hash {
         Hash::new(self.to_bytes())
     }
+
+    /// Number of fields included in the header.
+    ///
+    /// Fields instantiated with `None` values are excluded from the count.
+    pub(crate) fn len(&self) -> usize {
```

Comment:
Ah, sorry, missed this. I thought we decided to change this to `field_count()`, I think that works well.

Reply:
All sorted.